tests/test_resource_group.py

```diff
@@ -0,0 +1,23 @@
+from hypothesis import given, example
+from hypothesis.strategies import fixed_dictionaries, text, from_regex, uuids
+
+from cli.utils import RG_NAME_PATTERN, LOCATION_PATTERN
+from cli.validation import validate_uuid, validate_rg_name, validate_location
+
+@example(fixed_dictionaries({}), text(), 'a2cfad07-90d4-4e4c-9227-095d90fcc7dd')
+@given(fixed_dictionaries({}), text(), uuids(version=4))
+def test_validate_uuid(context, param, value):
+    assert validate_uuid(context, param, value)
+
+@example(fixed_dictionaries({}), text(), 'rg_name')
+@example(fixed_dictionaries({}), text(), 'rg.name')
+@example(fixed_dictionaries({}), text(), 'rg-name')
```

Comment:
should these be different values? (i honestly dont know)

Reply:
These are different by the `.`, `_`, `-`. I'll change the names a bit more so it's more clear to the user.